src/Core/src/Fonts/IFontRegistrar.cs

```diff
@@ -9,7 +9,7 @@ public interface IFontRegistrar
 
 		void Register(string filename, string? alias);
 
-		//TODO: Investigate making this Async
-		(bool hasFont, string? fontPath) HasFont(string font);
+		// TODO: this should be async as it involves copying files
+		string? GetFont(string font);
```

Comment:
There is NO reason to have these two in a tuple - we literally always have false,null and true,notnull. I even had an extension method that converted from a tuple into a single value... I am not sure my original thought process - might have been for 100% backwards compat. But that is not worth it.